### ISSUE TITLE
Include img alt text

### ISSUE THREAD

**Issue by gRegorLove** [indieweb/this-week#23]
When an event photo has alt text ([example](https://events.indieweb.org/2022/05/homebrew-website-club-pacific-JGTF0RmSRvLF), [example](https://events.indieweb.org/2022/06/homebrew-website-club-pacific-KS8OCRWFhheS)), include it in the generated newsletter ([example ](https://indieweb.org/this-week/2022-05-20.html) missing alt)
